pkg/email/email_test.go

```diff
@@ -0,0 +1,216 @@
+package email_test
+
+import (
+	_ "embed"
+	"errors"
+	"math"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/treeverse/lakefs/pkg/email"
+	"github.com/treeverse/lakefs/pkg/testutil"
+	"golang.org/x/time/rate"
+	"gopkg.in/gomail.v2"
+)
+
+var (
+	//go:embed invite_user.golden
+	inviteUserGolden string
+
+	//go:embed reset_password.golden
+	resetPasswordGolden string
+)
+
+type Dialer struct {
```

Comment:
MockDialer will make the code easy to understand

Reply:
Done